packages/wallet/src/components/AcknowledgeX.tsx

```diff
@@ -39,13 +34,9 @@ export default class AcknowledgeX extends React.PureComponent<Props> {
 const styles = StyleSheet.create({
   buttonContainer: {
     display: 'flex',
-    justifyContent: 'center',
```

Comment:
Probably beyond the scope of this change but we should move the styling into  a `.scss` file.